Bottom: 85d5387879ad7f7fdbe6f637414c4748dc685cc5
Top:    dbc6a9e87e9333f1a1acc7e0e23ec4743c2df930
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 17:58:08 +0800

Refresh of autofs-5.0.7-add-short-host-name-standard-marco-variable.patch

---

diff --git a/man/autofs.5 b/man/autofs.5
index 942b151..d2e7b12 100644
--- a/man/autofs.5
+++ b/man/autofs.5
@@ -170,7 +170,7 @@ UID	The user login ID
 GROUP	The user group name
 GID	The user group ID
 HOME	The user home directory
-SHOST	Short hostname (domainname removed if present)
+SHOST	Short hostname (domain part removed if present)
 .fi
 .RE
 .sp
